lib/google/ads/google_ads/google_ads_client.rb

```diff
@@ -69,12 +70,11 @@ module Google
   module Ads
     module GoogleAds
       class GoogleAdsClient
-        DEFAULT_API_VERSION = :V1
-        KNOWN_API_VERSIONS = [:V0, :V1]
 
         DEFAULT_CONFIG_FILENAME = 'google_ads_config.rb'
 
-        attr_reader :logger
+        attr_accessor :logger
+        attr_writer :lookup_util
```

Comment:
I'm not sure why this has to exist.

Reply:
the reason is the following: I intend to create a lookup object to create operations.

Then later, I intend to create the client.

Because I dont want to lose all the memoization done by the lookup object, I'd like to pass it to the client.

Reply:
Can you just add a comment here explaining that, and then the rest of this PR looks good. Sorry the review is taking so long; this is the first substantial PR from a community member :)

Reply:
done! I totally get some time is necessary to include contributions :)

Reply:
Can you clean up to:

    # Allow setting the lookup_util manually for users who use it before creating the client.

Thanks!

Reply:
done ;) bonus point: it allows caching the lookups but I'd rather not mention this.